tests/kiwi_images_test/validate_build.pm

```diff
@@ -0,0 +1,78 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: validate kiwi online build status from buildsystem
+# Maintainer: Ednilson Miura <emiura@suse.com>
+
+use base "installbasetest";
+use strict;
+use testapi;
+use utils;
+use version_utils "is_sle";
+my $sle_version = get_var('VERSION');
+my @files_list  = ('build.log');
+
+# check kiwi build status
+sub check_status {
+    my @builds = ("test-image-docker", "test-image-iso", "test-image-oem", "test-image-pxe", "test-image-pxe");
+    script_run("curl -k https://build.suse.de/project/monitor/QA:Maintenance:Images:$sle_version:kiwi-ng-testing > kiwi_out.html");
+    # sle12sp3 has 2 different builds
+    if (is_sle('<=12-SP3')) {
+        script_run("curl -k https://build.suse.de/project/monitor/QA:Maintenance:Images:$sle_version:kiwi-testing > kiwi_out2.html");
+        push @files_list, 'build2.log';
+    }
+    # parse build.log
+    foreach my $kiwi_build (@builds) {
```

Comment:
This could be made simpler with use of array and variable instead `build.log` and `build2.log` with just `build.log` value. `@builds` array would be different base on sle version.
Something like
```
my @builds = is_sle('<=12-SP3') ? ("test-image-oem") :  ("test-image-docker", "test-image-iso", "test-image-oem", "test-image-pxe", "test-image-pxe");
my $file_list = 'build.log';
foreach my $kiwi_build (@builds) {
    script_run("(grep $kiwi_build kiwi_out.html | grep -q succeeded && echo \"SLE-$sle_version $kiwi_build PASSED\" || 
        echo \"SLE-$sle_version $kiwi_build FAILED\") >> $file_list");
}

# show contents before check
script_run("cat $file_list");
```

Reply:
After discussion and clarifying leave it as it is, sle12sp3 has to test both kiwi and kiwi-ng